redisdb/metadata.csv

```diff
@@ -24,6 +24,7 @@ redis.mem.rss,gauge,,byte,,Amount of memory that Redis allocated as seen by the
 redis.mem.used,gauge,,byte,,Amount of memory allocated by Redis.,-1,redis,mem used
 redis.net.clients,gauge,,connection,,The number of connected clients (excluding slaves).,0,redis,clients
 redis.net.commands,gauge,,command,,The number of commands processed by the server.,0,redis,net commands
+redis.net.commands.instantaneous,,command,second,The instantaneous number of commands processed by the server at check time.,0,redis,net commands
```

Comment:
from the redis documentation: `Number of commands processed per second` is better in my opinion.
Also you are missing the `gauge` after the metrics name.